nix/default.nix

```diff
@@ -2,28 +2,30 @@
 , crossSystem ? null
 , config ? {}
 , sourcesOverride ? {}
+, gitrev ? null
```

Comment:
I don't see it used anywhere, is it really needed?

Reply:
Not used.  Removed.  Thanks!